src/rng.cpp

```diff
@@ -76,5 +76,7 @@ int djb2_hash( const unsigned char *str )
 double rng_normal( double lo, double hi )
 {
     static std::default_random_engine eng;
-    return std::normal_distribution<double>( ( hi + lo ) / 2, 3 )( eng );
+    double range = ( hi + lo );
+    double val = std::normal_distribution<double>( range / 2, range / 4 )( eng );
```

Comment:
No, the casting should be more explicit.
It isn't just "weighted `rng`". The distribution of values isn't monotonous - the clamping gives 2 extremes 2.5% chance each, meaning that for a wide enough range, those two are the most likely values.
It shouldn't be used for things like array indices.